domains/gc.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "gc.is-a.dev",
+  "owner": {
+    "username": "Gustavo",
```

username does not match github username
```suggestion
    "username": "gcjbr",
```